calico-cloud/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,576 @@
+Monitor the {{prodname}} Typha, Felix, and policy component metrics to ensure optimal cluster operation.
+
+## Concepts
+
+{{prodname}} Typha, Felix, and policy components are the most critical to monitor because they are responsible for ensuring networking and security functions are up-to-date and working as expected.
+
+### Typha
+
+Typha is a caching datastore proxy that sits between calico-nodes and Kubernetes API Server. Its primary function is to allow for increased cluster scale by reducing the load on Kubernetes API Server. Without Typha, large clusters (200+ nodes) would need a considerable amount of memory to correspond to the continuous watches and requests from calico-nodes running in the cluster.
+
+Typha maintains a single datastore connection on behalf of all of its clients (processes running in the calico-node pods, with Felix being Typhas’ main client). Typha watches for node, pod, network policy, bgp configuration, and other events on the Kubernetes API Server, caches and deduplicates this data, and fans out these events to its clients.
+
+### Felix
+
+Felix is a component of calico-node and is responsible for {{prodname}} network policy.
+Felix must be continuously in sync with the datastore to ensure the correct set of policies are applied to the node it is running on.
+
+![Typha-felix](/img/calico-enterprise/typha-felix.png)
+
+### About metrics
+
+Each {{prodname}} component that you want to connect to Prometheus for endpoint metrics has its own configuration (bgp, license, policy, felix, and typha).
+
+Note that Felix is a separate application with metric endpoints, its own core metrics to monitor itself, and a separate port for a second policy metric endpoint.
+
+## Metrics
+
+This section provides metrics recommendations for maintaining optimal cluster operations. Note the following:
+
+- Threshold values for each metric depend on the cluster size and churn rate.
+- Threshold recommendations are provided where possible, but because each cluster is different, and metrics can depend on cluster churn rate and scale. We recommend that you baseline the cluster to establish numbers that represent normal figures for your cluster.
+- Metrics that start increasing rapidly from the baseline set need attention.
+
+**Policy**
+- [{{prodname}} policy metrics](#calico-enterprise-policy-metrics)
+
+**Typha**
+- [Typha general metrics](#typha-general-metrics)
+- [Typha cluster mesh metrics](#typha-cluster-mesh-metrics)
+- [Typha client metrics](#typha-client-metrics)
+- [Typha cache internals](#typha-cache-internals)
+- [Typha snapshot details](#typha-snapshot-details)
+
+**Felix**
+- [Felix cluster state metrics](#felix-cluster-state-metrics)
+- [Felix error metrics](#felix-error-metrics)
+- [Felix time-based metrics](#felix-time-based-metrics)
+
+## Policy metrics
+
+The following policy metrics are used in Manager UI and requires special Prometheus configuration to scrape the metrics. See [Policy metrics](./policy-metrics) for help.
```

Comment:
I'd use a note box just to flag that this is a separate endpoint exposed by felix (and details are in the link).

Reply:
These metrics are exposed by Felix so it seems a little odd to have felix metrics, then Typha metrics, then Felix again?

Reply:
Added note and moved Policy metrics under Felix section.